src/redprl/refiner.fun

```diff
@@ -245,7 +245,12 @@ struct
         val motiven = substVar (n, x) motiveHole
         val motivem = substVar (m, x) motiveHole
 
-        val (H', catjdg') = Selector.map sel (fn _ => motiven) (H, catjdg)
+        fun replace jdg = 
+          case jdg of 
+             AJ.TRUE (_, l) => AJ.TRUE (motiven, l)
+           | _ => jdg
+
+        val (H', catjdg') = Selector.map sel replace (H, catjdg)
```

Comment:
Are you saying that the map now accidentally replaces the `l` due to the new fix about level substitution?

Reply:
@favonia Yes; but it was probably not right before that even, because `Selector.map` was calling `AJ.map`; when the judgment in question is an equality `m = n in ty`, it would replace it with `motiven = motiven in motiven`.

Reply:
Well, it can and should only rewrite a `TRUE` judgment.

Reply:
The case analysis above on `Selector.lookup` rejects other forms of judgments.